engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryImpl.java

```diff
@@ -75,36 +76,9 @@
 	private static final Log LOG = LoggerFactory.make( MethodHandles.lookup() );
 
 	/**
-	 * The default message interpolator for this factory.
+	 * Context containing all {@link HibernateValidatorFactory} level helpers and configuration properties.
```

Comment:
I think we can refer to the generic `ValidatorFactory` here. It's more to compare with `Validator` or validation-scoped elements.